lib/declarations.d.ts

```diff
@@ -690,6 +714,11 @@ interface IPluginInfoBase {
 	 * Plugin specific identifier. It can differ from plugin's name.
 	 */
 	Identifier: string;
+
+	/**
+	 * Variables, that have to be set in the application in order to be able to use the plugin correctly.
+	 */
+	Variables?: string[]|IDictionary<INativeScriptPluginVariable>;
```

Comment:
not sure, but I've just moved it from one inteface to another. If we decide to change it, we'll have to manually check the code (just renaming does not work :cry: ). Do you want me to do it in this PR?